grid-client/deployer/tf_plugin_client.go

```diff
@@ -95,20 +95,81 @@ type TFPluginClient struct {
 	cancelRelayContext context.CancelFunc
 }
 
+type pluginCfg struct {
+	substrateURL  []string
+	relayURL      string
+	proxyURL      string
+	rmbTimeout    int
+	showLogs      bool
+	rmbInMemCache bool
+}
+
+type PluginOpt func(*pluginCfg)
+
+func WithSubstrateURL(substrateURL []string) PluginOpt {
```

Comment:
```suggestion
func WithSubstrateURL(substrateURL ...string) PluginOpt {
```

would that be better so user can pass one or more without wrapping it in a slice.
so both can work: `WithSubstrateURL("url")` and `WithSubstrateURL(someSlice...)` instead of `WithSubstrateURL([]string{"url"})`